dymos/utils/command_line.py

```diff
@@ -75,6 +77,7 @@ def dymos_cmd(argv=None):
         'restart': args.solution,
         'simulate': args.simulate,
         'no_solve': args.no_solve,
+        'no_iterate': args.no_iterate,
```

Comment:
Note:  In the future no_iterate will not be necessary once OpenMDAO's recording structure is revamped.  For now we need it in order to get results from the driver recorder.